Tensile/KernelWriterAssembly.py

```diff
@@ -6844,7 +6844,7 @@ def openSumAtLeastUnroll(self, kernel, prefetch, isOptNLL, isPap):
         return ""
 
       skipOptNLL = self.getNamedLabel("OptNLL_End")
-      tmpSgpr = self.getTmpSgpr(2).idx()
+      tmpSgpr = self.getTmpSgpr(4).idx()
```

Comment:
Please make sure rocblas build works.

Reply:
rocblas build worked, and gemm tests ran overnight on gfx90a